medinify/sentiment/review_classifier.py

```diff
@@ -8,13 +8,23 @@
 import string
 import math
 import csv
```

Comment:
It looks like these changes are from when you were going to add the NN to directly to this class. Can you remove them or talk to me about why they're here?

Reply:
(This goes for the whole file)